File: Patterns/2. Rectangle Patterns/17. Rectangle Binary Pattern/ron.cpp
#include <iostream>
using namespace std;
int main(){
    int l,b;
    cout<<"Enter the Length : ";
    cin>>l;
    cout<<"Enter the Breadth : ";
    cin>>b;

    for(int i=1; i<=l; i++){
        for(int j=1; j<=b; j++){
            cout<<j%2<<" ";
        }
        cout<<'\n';
    }
}

/*
Output

1 0 1 0 1 0 1 
1 0 1 0 1 0 1 
1 0 1 0 1 0 1 
1 0 1 0 1 0 1 
1 0 1 0 1 0 1 

*/

File: Patterns/2. Rectangle Patterns/15. Rectangle Odd Numbers/ron.py
l = int(input("Enter the Length : "))
b = int(input("Enter the Breadth : "))

for i in range(1,l+1):
    for j in range(1,b+1):
            print(2*j-1, end=' ')
    print()

'''
Output:

1 3 5 7 9 11 13 
1 3 5 7 9 11 13 
1 3 5 7 9 11 13 
1 3 5 7 9 11 13 
1 3 5 7 9 11 13    
'''